### ISSUE TITLE
ATC-Code fehlt

### ISSUE THREAD

**Issue by zdavatz** [zdavatz/oddb2xml#25]
For example: 
_Viagra 50 mg, Filmtabletten_ **has no ATC-Code**
_VIAGRA Filmtabl 50 mg 4 Stk_ **has the ATC-Code**

Cloud you please explain why is this happening? This can be seen again and again. Here you can find another example: 
```
<PRD DT="" SHA256="4c6bfb78cd3549cc85e334c4e87fb9f23ec64cbc7b2ccfbe67e023d44c548a46">
   <GTIN>7680002780020</GTIN>
   <PRODNO>0027801</PRODNO>
   <DSCRD>Colon Sérocytol, suppositoire</DSCRD>
   <DSCRF>Colon Sérocytol, suppositoire</DSCRF>
   <IT>08.07.</IT>
   <CPT/>
   <PackGrSwissmedic>9</PackGrSwissmedic>
   <EinheitSwissmedic>Suppositorien</EinheitSwissmedic>
   <SubstanceSwissmedic>globulina equina (immunisé avec tissu intestinal porcin)</SubstanceSwissmedic>
   <CompositionSwissmedic>globulina equina (immunisé avec tissu intestinal porcin) 8 mg, propylenglycolum, conserv.: E 216, E 218, excipiens pro suppositorio.</CompositionSwissmedic>
 </PRD>
```

**Comment by zdavatz** [comment]
The products with no ATC-Code are not yet listed in Refdata. So that means that they are most probably not yet on the market in Switzerland.

Here we could just import the ATC-Code from Swissmedic.

**Comment by zdavatz** [comment]
For you that just means, that you can ignore any GTIN that starts with
7680 and does not have any ATC-Code because these products will not be
for sale on the Swiss market anyway.

All GTIN's of pharmaceutical drugs start with 7680 in Switzerland. 76
stands for Switzerland and 80 stands for Swissmedic.

**Comment by ngiger** [comment]
I found the culprit. We never emitted an ATC code when the package was only in the swissmedic packungen but not in refdata. Fixed with 2.5.7

**Comment by zdavatz** [comment]
Great, thank you!
